src/views/catalog/CreateFromInstanceTypes/components/BootableVolumeList/BootableVolumeList.tsx

```diff
@@ -109,7 +109,7 @@ const BootableVolumeList: FC<BootableVolumeListProps> = ({
 
   useEffect(() => {
     if (!isAdmin && volumeListNamespace === ALL_PROJECTS) {
-      setVolumeListNamespace(OPENSHIFT_OS_IMAGES_NS);
+      setVolumeListNamespace(isUpstream ? KUBEVIRT_OS_IMAGES_NS : OPENSHIFT_OS_IMAGES_NS);
```

Comment:
This seems like something we should wrap in a hook like `useOSImagesNS` instead of doing it manually in every place that we need it. Thoughts?